bundles/org.openhab.binding.dbquery/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,125 @@
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="query">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="influxdb2"/>
+			<bridge-type-ref id="jdbc"/>
+		</supported-bridge-type-refs>
+		<label>Query Thing</label>
+		<description>Thing that represents a native query</description>
+
+		<channels>
+			<channel id="execute" typeId="execute-channel"/>
+			<channel id="resultString" typeId="result-channel-string"/>
+			<channel id="resultNumber" typeId="result-channel-number"/>
+			<channel id="resultDateTime" typeId="result-channel-datetime"/>
+			<channel id="resultContact" typeId="result-channel-contact"/>
+			<channel id="resultSwitch" typeId="result-channel-switch"/>
+
+			<channel id="parameters" typeId="parameters-channel"/>
+			<channel id="correct" typeId="correct-channel"/>
+			<channel id="calculateParameters" typeId="calculate-parameters-channel"/>
+		</channels>
+
+
+
+		<config-description>
+			<parameter name="query" type="text" required="true">
+				<label>Query definition</label>
+				<description>Query definition using native query language</description>
+				<context>script</context>
+			</parameter>
+			<parameter name="hasParameters" type="boolean">
+				<label>Query has parameters</label>
+				<description>True if the query has parameters, otherwise false</description>
+				<default>false</default>
+			</parameter>
+			<parameter name="scalarResult" type="boolean">
+				<label>Scalar result</label>
+				<description>True if the query always return only one single scalar value (only one row and one value-column in this
+					row), otherwise false</description>
+				<default>true</default>
+			</parameter>
+			<parameter name="scalarColumn" type="text" required="false">
+				<label>Scalar column name</label>
+				<description>The column's name that is used to extract scalarResult. If only one column is returned this
+					parameter
+					can be blank</description>
+			</parameter>
+			<parameter name="interval" type="integer" min="0">
+				<label>Interval</label>
+				<description>
+					An interval, in seconds, the query will be repeatedly executed. Default values is 0, which means that
+					query is never executed manually (you need to send ON to execute channel)
+				</description>
+				<default>0</default>
+			</parameter>
+			<parameter name="timeout" type="integer" min="0">
+				<label>Timeout Query</label>
+				<description>
+					A time-out in seconds to wait for the query result, if it's exceeded result will be discarded.
+					Use 0 for
+					no timeout
+				</description>
+				<default>0</default>
+			</parameter>
+
+
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="execute-channel">
+		<item-type>Switch</item-type>
+		<label>Executed query channel</label>
```

Comment:
```suggestion
		<label>Executed Query</label>
```
labels should be:
1. Short, less than 25 characters.
2. Each word starts in capitals unless it is a joining word like or, and, a ...
3. It will get used to auto name the items with when the framework does the feature "creates equipment from a thing".